internal/pkg/daemon/bpfrecorder/generate/main.go

```diff
@@ -189,16 +189,17 @@ func generateBtf(builder *strings.Builder) error {
 			fmt.Fprintf(docs, "- %s\n", os)
 		}
 		if _, ok := btfs[os][osVersion]; !ok {
+			const indent = 2
 			btfs[os][osVersion] = map[types.Arch]map[types.Kernel][]byte{}
-			fmt.Fprintf(docs, "%s- %s\n", strings.Repeat(" ", 2), osVersion) // nolint:gomnd // indent
+			fmt.Fprintf(docs, "%s- %s\n", strings.Repeat(" ", indent), osVersion) //olint:gomnd // indent
```

Comment:
`olint` sounds like a typo?

Reply:
Yeah this one can be removed completely. :+1: